File: 3. Conditional Loops/1. Cpp/1. Basic/maxandmintillzero.cpp
#include <bits/stdc++.h>
using namespace std;
#define ll long long
#define FIO ios_base::sync_with_stdio(false), cin.tie(0), cout.tie(0)
#define pi 3.14
#define endl "\n"

int main() {
    FIO;
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
    int n;
    //cout << "How many numbers you want to enter: " << endl;
    cin >> n;
    int max = INT_MIN;
    int min = INT_MAX;
    for (int i = 0; i < n; ++i) {
        int num;
        //cout << "Enter number -" << i + 1 << ": ";
        cin >> num;
        if (num > max) max = num;
        if (num < min) min = num;
    }
    cout << "Maximum number is: " << max << endl;
    cout << "Minimum number is: " << min << endl;
    return 0;
}

File: 3. Conditional Loops/2. Java/1. Basic/maxandmintillzero.java
import java.util.Scanner;
public class maxandmintillzero {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int n;
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        System.out.println("ENTER THE NUMBERS YOU WANT TO ENTER: ");
        n = sc.nextInt();
        for(int i=1;i<=n;i++){
            System.out.println("Enter number -  " + i + ": ");
            int num = sc.nextInt();
            if(num > max) max = num;
            if(num < min) min = num;
        }
        System.out.println("Maximum number is: " + max);
        System.out.println("Minimum number is: " + min);
        sc.close();
    }    
}
